Strict Parsing Decimal Documentation (#1524)
AbsaOSS/enceladus: _docs/2.0.0/usage/schema.md

| [grouping_separator](#grouping_separator) | any numeric type            | Character to mark boundaries between orders of magnitude, usually to mark thousands, millions etc.                                                    | *\_*            | *,*                                          |
| [minus_sign](#minus_sign)                 | any numeric type            | Character to mark the number is negative.                                                                                                             | *N*             | *-*                                          |
| [allow_infinity](#allow_infinity)         | float & double              | Flag indicating if the column accepts infinity as a value (and positive/negative numbers which are too large are converted to *infinity*/*-infinity*) | *true*          | *false*                                      |

> | [strict_parsing](#strict_parsing)         | decimal                     | Flat indicating strict parsing on the provided decimal value. Strict parsing rejects a value(including default) with a larger scale than allowed      | *true*          | *false*                                      |

Reviewer: ```suggestion
| [strict_parsing](#strict_parsing)         | decimal                     | Flag indicating if strict parsing should be done on the input value. Strict parsing rejects a value with a more decimal value than the defined scale of the decimal       | *true*          | *false*                                      |
```

> | [strict_parsing](#strict_parsing)         | decimal                     | Flag indicating if strict parsing should be done on the input value. Strict parsing rejects a value with a higher decimal value the defined scale     | *true*          | *false*                                      |

Reviewer: ```suggestion
| [strict_parsing](#strict_parsing)         | decimal                     | Flag indicating if strict parsing should be done on the input value. Strict parsing rejects a value with more decimal places than the defined scale     | *true*          | *false*                                      |
```
Sorry, the previous suggestion didn't make sense there.

The string representing infinity on input is *"∞"* and *"-∞"* respectively. Positive and negative numbers with values 
that are too large are converted to *infinity* and *-infinity*, respectively.

### strict_parsing

<sup>Supported by types:** [Decimal](#decimal)</sup>

> Flat indicating strict parsing on the provided decimal value.

Author: `Exercised` sounds weird for me. I would say `applied`

> Flat indicating strict parsing on the provided decimal value.

Reviewer: Applied it good 👍

> Strict parsing rejects a value(including default) with a larger scale than allowed with casting error
> ([see here]({{ site.baseurl }}/docs/{{ page.version }}/usage-errcol)). For example for Decimal(X,2), the values with
> longer scale(like 10.12345, 0.1234) will rejected 

Reviewer: ```suggestion
Strict parsing rejects a value with more decimal places than the scale (second number) of the field's Decimal type definition. 
This results in a casting error in error column ([see here]({{ site.baseurl }}/docs/{{ page.version }}/usage-errcol)). For example for Decimal(X,2), the values with
longer scale (like 10.12345, 0.1234) will be rejected .
Default value have to fit the scale as well in the case of strict parsing.
```

cannot be a `default` for the type [`Short`](#short), or _"∞"_ if `allow_infinity` is _"false"_ for [`Double`](#double)/
[`Float`](#float))
- If it's a type supporting [`pattern`](#pattern) and it is defined, the default value has to adhere to the `pattern`

> - If 'strict_parsing' is enabled for [`Decimal`](#decimal), the decimal scale should fit into the specified type scale

Reviewer: ```suggestion
- If 'strict_parsing' is enabled for [`Decimal`](#decimal), the number of decimal places has to fit into the type's scale
```
